docker-compose.test.yml

```diff
@@ -77,3 +77,9 @@ services:
       - $HOME/.gnupg:/root/.gnupg:ro
     working_dir: /opt/raster-foundry/app-backend/
     entrypoint: ./sbt
+
+  lambda:
+    image: python:3.6
```

Comment:
would it make more sense to use something like lambci/lambda:python3.6, for consistency with the deployment environment?

Reply:
I tried this and got an error - I think it makes sense if we want to test running the zip file, but building the zip file I think we don't need and maybe shouldn't since some utilities and dependencies aren't there for installing the packages.